terraform/alerting/module.availability-slo/main.tf

```diff
@@ -22,6 +22,7 @@ resource "google_monitoring_slo" "slo" {
   slo_id       = "availability-slo-${var.service_name}"
   display_name = "${var.goal * 100}% of requests are successful over rolling 28 days (service=${var.service_name})"
   project      = var.project
+  count        = var.enabled ? 1 : 0
```

Comment:
Nit, count typically comes at the very top of the resource.

Reply:
Fixed.